.shellcheckrc

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+# shellcheck disable=all
```

Comment:
It seems to not be working: 

![image](https://user-images.githubusercontent.com/1157864/233401181-23d7ced7-27b8-4c80-8fe0-e062ca976076.png)

Maybe you should remove the comment:

```suggestion
shellcheck disable=all
```

But the issues seem to be easy to fix, feel free to call me if you want me to help you to fix it, it would take us about ten minutes.

Reply:
will do! thanks

Reply:
Thanks a lot @andresmoschini!! ❤️